StdUi.lua

```diff
@@ -295,3 +295,55 @@ function StdUi:MakeResizable(frame, direction)
 	end)
 end
 
+--- Get a StdUi font object.
+--- This method generates font objects on demand at runtime and caches them for reuse.
+--- Each font is classified by a class and a color code.
+--- Classes are defined by the application via StdUi.config.fontClasses{}. The default class is 'default'.
+--- Color codes are defined by the application via StdUi.config.font.color{}. The default color is 'normal'.
+--- @usage font = StdUi:GetFontObject('scrollingMessage', 'normal')
+--- @usage font = StdUi:GetFontObject('default', 'highlight')
+--- @usage font = StdUi:GetFontObject('default', 'disabled')
+--- @param class string A key of StdUi.config.fontClasses, or 'default'
+--- @param colorCode string A key of StdUi.config.font.color ('normal' by default)
+function StdUi:GetFontObject(class, colorCode)
+	class = class or 'default'
+	colorCode = colorCode or 'normal'
+
+	-- initialize self._RUNTIME_FONTS
+	self._RUNTIME_FONTS = self._RUNTIME_FONTS or {}
```

Comment:
Please don't use underscores and in general this type of variable naming. All other cache-type variables are camelCase or sometimes PascalCase. I belive it should be just named `runtimeFonts`

Reply:
Fixed locally, will push.